bucket/v2rayn.json

```diff
@@ -16,6 +19,9 @@
         "if (!(Test-Path \"$persist_dir\\config.json\")) {",
         "    Copy-Item \"$(appdir v2ray $global)\\current\\config.json\" \"$dir\" | Out-Null",
         "}",
+        "if (!(Test-Path \"$persist_dir\\config.json\") and (Test-Path \"$(appdir xray $global)\\current\\config.json\")) {",
```

Comment:
This will be always true, since xray is automatically installed

Reply:
But If the user does not manually add config.json for xray, it wont be generated

Reply:
That file should be persisted in xray, otherwise it does not make sense as after xray update and then v2ray update it will not be working.

Reply:
I removed it.